gatsby-config.js

```diff
@@ -1,5 +1,9 @@
 const path = require('path');
 
+require('dotenv').config({
+  path: `.env.${process.env.NODE_ENV}`,
```

Comment:
Just to confirm, we are using `.env.dev` and `.env.prod`, right?

Reply:
It wasn't, I just added a commit that does so.